### ISSUE TITLE
[MAS1.4.1] Links present in 'Welcome' pane are identifiable as link by their color only.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#737]
**Actual Result:**
The only way to know the difference between links and normal text present on 'Welcome' screen is Blue color of those links.

**Expected Result:**
There should be some other way as well to know the difference between links and normal text present on 'Welcome' screen beside Blue color of those links. All links must be underlined.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome screen will open up.
3. Notice links present on this page (In Blue).

**User Impact:**
Users with low vision or color-blindness will find it difficult to identify these links.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 1.4.1 – Use of Color (16)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={9f1f0271-42a1-45b1-be00-31473ba2e579})

**Attachments:**
![mas1 4 1_botframeworkemulator_welcomepage_links](https://user-images.githubusercontent.com/41668935/43773318-faa9e4b2-9a62-11e8-8e3a-7176f503e0f0.png)

**Comment by corinagum** [comment]
@mewa1024 Could you let me know how we want to update this one? Should the links have underlines, or something else? Thank you.

**Comment by mewa1024** [comment]
@corinagum Underline for now.
